build-system/babel-plugins/babel-plugin-transform-jss/index.js

```diff
@@ -126,3 +127,21 @@ module.exports = function ({types: t, template}) {
     },
   };
 };
+
+// Abuses spawnSync to let us run an async function sync.
+function transformCssSync(cssText) {
+  const programText = `
+    const {transformCss} = require('../../../build-system/tasks/jsify-css');
+    transformCss(\`${cssText}\`).then((css) => console.log(css.toString()));
+  `;
+
+  // TODO: migrate to the helpers in build-system exec.js
+  // after adding args support.
+  const spawnedProcess = spawnSync('node', ['-e', programText], {
+    cwd: __dirname,
+    env: process.env,
+    encoding: 'utf-8',
+    stdio: 'pipe',
+  });
+  return spawnedProcess.stdout;
```

Comment:
We should be checking the return code and `stderr` to see if any errors happened.

Reply:
Good thinking. If anything is written to `stderr` or if there is a bad exit code we'll want to throw.

Reply:
> We should be checking the return code and stderr to see if any errors happened.

Ok, so we actually shouldn't use `stderr` to determine failure, since even warnings go there.

Reply:
Yah, this is the correct way to do it.